persistence/src/commonMain/db_user/migrations/32.sqm

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE Message ADD COLUMN expire_after_millis INTEGER DEFAULT(NULL);
+ALTER TABLE Message ADD COLUMN self_deletion_start_date INTEGER AS Instant DEFAULT(NULL);
```

Comment:
I guess this is unintentional in this PR?

PR adding tests, not modifying database. Also seems that the migration verification isn't passing.

Reply:
It was not added in a previous PR so I assume we need to add it ?

Reply:
I see that the [previous PR](https://github.com/wireapp/kalium/pull/1552) was merged without passing all the tests.

![image](https://user-images.githubusercontent.com/9389043/227892166-69c43524-ef6f-4988-92dc-d62634c1e2a4.png)

Please be careful when merging to an epic branch, as these do not have the `required` checks like `develop` and other fixed branches have. So, we need to look at the checks before hitting the _merge_ button more than in regular PRs, and merging without passing the checks just complicates everything unnecessarily, making the first PR look broken, and the following PR messier.

----

In any case. This migration is not complete: `MessageDetailsView` must be recreated too.

I'd advise to create a PR just with this migration fixes and merge it to the epic branch. Making it atomic, parallelising reviewers and making sure we leave breadcrumbs of the work being done.

A PR with 4 lines of migration code, explaining that these were missed from the previous PR, and that checks are now passing: _every reviwer's dream_.

Reply:
created a seperate PR : https://github.com/wireapp/kalium/pull/1599